src/get-installation-authentication.ts

```diff
@@ -16,6 +16,12 @@ export async function getInstallationAuthentication(
   const installationId = (options.installationId ||
     state.installationId) as number;
 
+  if (typeof installationId !== "number") {
+    throw new Error(
+      "installationId is required for installation authtentication."
```

Comment:
typo in "authtentication" :) 

I like to prefix error messages with the package names. And maybe clarify that `installationId` is an option? How about this

```suggestion
      "[@octokit/auth-app] installationId opion is required for installation authentication."
```

Reply:
Oops haha, fix a typo/leave a typo I guess! That looks pretty good to me, I'll update that and the test for it as well.